addons/mass_mailing/views/unsubscribe_templates.xml

```diff
@@ -71,6 +71,19 @@
                         <p>You have been successfully <strong>unsubscribed</strong>!</p>
                     </div>
 
+		    <div t-if="list_ids" id="blacklist_warning" class="alert alert-warning">
+		      <p class="text-center">You were subscribed to those newsletters, you won't receive them anymore:</p>
```

Comment:
I would use the present tense, something like: "You are currently subscribed to those newsletters. Unsubscribing means you will not receive any news from them anymore : ..." . Concerning "won't" is used more when speaking but not when writing I think.